packages/map-template/src/components/Directions/Directions.jsx

```diff
@@ -189,11 +171,15 @@ function Directions({ isOpen, onBack, onSetSize, snapPointSwiped }) {
     /**
      * Close the directions.
      * Reset the active steps and stop rendering directions.
+     * Set the floor according to the current location's floor.
      */
     function onDirectionsClosed() {
         resetSubsteps();
         stopRendering();
         onBack();
+
+        const currentFloor = mapsIndoorsInstance.getFloor();
+        mapsIndoorsInstance.setFloor(currentFloor);
```

Comment:
It is confusing why this is necessary: We read out the current floor and sets that floor to be the current floor 🤔

Reply:
That is true, removed the unnecessary call for setting the floor. ✅